build/node.json

```diff
@@ -4,7 +4,7 @@
   "metadata": {
     "name": "fb4ebb70-2783-42b8-b3ef-63e2fd6d242e",
     "labels": {
-      "name": "edge-node"
+      "node-role.kubernetes.io/edge":"edge"
```

Comment:
do we need to add any steps in documentation to specify that this new label needs to be added for an edge node ?
also , what is the behaviour if it's empty etc ? what if a user add's anything else as a role other than "edge" , it won't be considered as an edge node ? please clarify in readme about this.